src/OpenTK/Platform/Linux/LinuxNativeWindow.cs

```diff
@@ -374,6 +374,11 @@ public override Point PointToScreen(Point point)
             return new Point(point.X + client.X, point.Y + client.Y);
         }
 
+        public override void ConfineCursor (bool confine)
+        {
+            throw new NotImplementedException ();
```

Comment:
Linux supported through SDL and X11. Honestly I dont know much about LinuxNativeWindow. Last time I tried to create it I get errors.